Dockerfile

```diff
@@ -1,29 +1,29 @@
 # syntax=docker/dockerfile-upstream:master
 
-ARG RUNC_VERSION=v1.3.3
-ARG CONTAINERD_VERSION=v2.1.4
+ARG RUNC_VERSION=v1.4.0
```

Comment:
Not sure if we should already go to 1.4, knowing that containerd v2.1 and v2.2 are still on v1.3.x; https://github.com/containerd/containerd/blob/b77253fbc2cf01be361f63a03d929ec53642fd59/script/setup/runc-version

Reply:
There was some discussion about a breaking change in runc 1.4, therefore it was not update yet in 2.2; not sure if that still applies;

- https://github.com/containerd/containerd/pull/12593#issuecomment-3598597472